CHANGELOG.md

```diff
@@ -18,7 +18,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add a workaround to avoid 480p resolution scale down when there are 5-8 videos for the default video uplink policy 
   for Chromium browsers version 98 on Windows and use 360p instead.
 - Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.
+
+## [2.28.0] - 2022-02-08
+
+### Added
+- Add compression support when sending and receiving sdp messages.
```

Comment:
Since we will be including this change with both releases, can you add the same changelog under beta-0 as well? Also, currently in release-2.x the changelog mentions 2.27.1, can you please update it to 2.28.0 and add the [one change](https://github.com/aws/amazon-chime-sdk-js/blob/release-2.x/CHANGELOG.md#fixed) there here as well?

Basically we initially dont know whether it is patch or a simple fix hence kept it at 2.27.1, with the SDP changes 2.27.1 will need to be changed to 2.28.0.

Reply:
Sorry for the confusion about this, can address this at the last but this is how the changelogs should look like for both the release-2.x vs main:
PR for release-2.x:

Remove below from changelog:
## [2.27.1] - 2022-02-17

### Added

### Removed

### Fixed
- Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.

Add:
## [2.28.0] - 2022-02-17

### Added
- Add compression support when sending and receiving sdp messages.

### Removed

### Changed

### Fixed
- Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.



PR for main:
## [3.0.0-beta.0] - 2022-02-08
    
### Added
- Add compression support when sending and receiving sdp messages.
    
### Removed
- Remove support for Plan B as well as Safari (and iOS) 12+. The minimum Safari and iOS supported version is now 13. Also clean up all plan-B code path.
- Remove all deprecated meeting status code.  
    
### Changed
- Add support for pre-release in Versioning.
- Removed upward BWE throttling logic in VideoPriorityBasedPolicyConfig, which was increasing recovery time more then intended, whereas its main focus was towards slowing downturns in BWE when the network is actually stable. We may come back to configuring the recovery delay another time.
- Add support for aws-sdk js v3 for messaging session.
    
### Fixed
- Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.
- Add a workaround to avoid 480p resolution scale down when there are 5-8 videos for the default video uplink policy 

## [2.28.0] - 2022-02-17

### Added
- Add compression support when sending and receiving sdp messages.

### Removed

### Changed

### Fixed
- Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.